osu.Game/Rulesets/Judgements/Judgement.cs

```diff
@@ -46,10 +46,14 @@ public class Judgement
 
         /// <summary>
         /// Whether the <see cref="Result"/> should affect the combo portion of the score.
```

Comment:
This shouldn't say `combo portion`, but reference whether the combo is increased or decreased.